src/Catty UITests/ObjectTVCTests.swift

```diff
@@ -39,111 +39,111 @@ class ObjectTVCTests: XCTestCase, UITestProtocol {
         let appTables = app.tables
         let projectObjects = ["Mole 1", "Mole 2", "Mole 3", "Mole 4"]
 
-        appTables.staticTexts["Continue"].tap()
+        appTables.staticTexts[kLocalizedContinue].tap()
 
         //check every mole for script
         for object in projectObjects {
             appTables.staticTexts[object].tap()
-            appTables.staticTexts["Scripts"].tap()
-            XCTAssert(app.navigationBars["Scripts"].buttons[object].exists)
-            app.navigationBars["Scripts"].buttons[object].tap()
-            app.navigationBars[object].buttons["My first project"].tap()
-            XCTAssert(app.navigationBars["My first project"].exists)
+            appTables.staticTexts[kLocalizedScripts].tap()
+            XCTAssert(app.navigationBars[kLocalizedScripts].buttons[object].exists)
+            app.navigationBars[kLocalizedScripts].buttons[object].tap()
+            app.navigationBars[object].buttons[kLocalizedMyFirstProject].tap()
+            XCTAssert(app.navigationBars[kLocalizedMyFirstProject].exists)
         }
     }
 
     func testScriptsCanDeleteBrickSetSizeTo() {
         let app = XCUIApplication()
 
-        app.tables.staticTexts["Continue"].tap()
+        app.tables.staticTexts[kLocalizedContinue].tap()
         app.tables.staticTexts["Mole 1"].tap()
-        app.tables.staticTexts["Scripts"].tap()
+        app.tables.staticTexts[kLocalizedScripts].tap()
 
         //delete the SetSizeTo brick
-        app.collectionViews.cells.element(boundBy: 1).staticTexts["Set size to "].tap()
-        app.buttons["Delete Brick"].tap()
+        app.collectionViews.cells.element(boundBy: 1).staticTexts[kLocalizedSetSizeTo + " "].tap()
```

Comment:
The whitespace solution is not ideal. In this case we don't need the static text to tap the cell. If you remove ".staticTexts[kLocalizedSetSizeTo + " "]", it still works. You can do a check above this line to see if it really is a SetSize brick.
Maybe you can also come up with a solution to get rid of the other whitespace occurences.

Reply:
![screenshot 2019-02-25 at 12 05 22](https://user-images.githubusercontent.com/4242986/53333386-02b46d80-38f6-11e9-9cee-89836dc5f60c.png)
You can use NSPredicate to query the staticTexts, see the example above. If you write a helper function for this case you can (re-)use it for all whitespace occurences.

Reply:
```suggestion
    func testScriptsCanDeleteBrickSetSizeTo() {
        let app = XCUIApplication()

        app.tables.staticTexts[kLocalizedContinue].tap()
        app.tables.staticTexts["Mole 1"].tap()
        app.tables.staticTexts[kLocalizedScripts].tap()

        //delete the SetSizeTo brick
        app.collectionViews.cells.element(boundBy: 1).staticTextStartsWith(kLocalizedSetSizeTo).tap()
        app.buttons[kLocalizedDeleteBrick].tap()
```

Catty UITests/Extensions/XCUIElement+staticTexts.swift
```suggestion
import XCTest

extension XCUIElement {
    func staticTextStartsWith(_ queryString: String) -> XCUIElement {
        return self.staticTexts
            .element(matching: NSPredicate(format: "label BEGINSWITH '"+queryString+"'"))
    }
}
```
Example how to use it as Extension